openwisp_firmware_upgrader/tests/test_openwrt_upgrader.py

```diff
@@ -34,6 +34,14 @@ def mocked_exec_upgrade_not_needed(command, exit_codes=None):
         f'test -f {OpenWrt.CHECKSUM_FILE}': ['', 0],
         f'cat {OpenWrt.CHECKSUM_FILE}': [TEST_CHECKSUM, 0],
     }
+    # Handle the UUID command dynamically
+    if command == 'uci get openwisp.http.uuid':
+        # Return the device UUID that matches the one in the test
+        from openwisp_firmware_upgrader.models import DeviceFirmware
```

Comment:
In OpenWISP, we don't import the model directly. We use the swapper module to load the model. 

The `DeviceFirmware` model is already [loaded globally](https://github.com/openwisp/openwisp-firmware-upgrader/blob/78652eb953c8e59363764b65f9a6ba548b63673e/openwisp_firmware_upgrader/tests/test_openwrt_upgrader.py#L24). You don't need to do this here.